core/src/main/java/org/mskcc/cbio/oncokb/apiModels/TumorType.java

```diff
@@ -13,15 +15,25 @@
 @ApiModel(description = "OncoTree Detailed Cancer Type")
 public class TumorType implements Serializable {
 
+    @ApiModelProperty(value = "Database TumorType ID")
     private Integer id = null;
+    @ApiModelProperty(value = "Oncotree code")
```

Comment:
Is it possible to include a link to oncotree?

Reply:
I can put a link in the ApiModel annotation